rpc/src/v1/types/account_info.rs

```diff
@@ -13,13 +13,44 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+use v1::types::{ H160, H256, U256, Bytes};
 
 /// Account information.
 #[derive(Debug, Default, Clone, PartialEq, Serialize)]
 pub struct AccountInfo {
 	/// Account name
 	pub name: String,
 }
+/// Datastructure with proof for one single storage-entry
```

Comment:
empty line between the end-parenthesis and doc comment, please

Reply:
fixed